MimeKit/GroupAddress.cs

```diff
@@ -111,7 +111,7 @@ public GroupAddress (string name) : this (Encoding.UTF8, name)
 		/// <returns>The cloned group address.</returns>
 		public override InternetAddress Clone ()
 		{
-			return new GroupAddress (Encoding, Name, Members.Select (x => x.Clone ()));
+			return new GroupAddress (Encoding, Name, Members.Select (static x => x.Clone ()));
 		}
```

Comment:
I've never seen static used that way. What does it mean in this context?

Reply:
It ensures that the method doesn't capture any outside variables and creating a closure / allocation. It also creates a new scope for the variable names. In this case, we weren't capturing any outside variables, so the codegen is the same.

Reply:
Ah, gotcha. Seems like a good feature to keep in mind. Thanks.